testscript/doc.go

```diff
@@ -150,6 +150,9 @@ The predefined commands are:
   test. At the end of the test, any remaining background processes are
   terminated using os.Interrupt (if supported) or os.Kill.
 
+  The standard input can be provided using the stdin command; this will be
```

Comment:
Superfluous "The" at the start of the sentence?

Reply:
done